tools/clang/test/CodeGenHLSL/Samples/MiniEngine/ParticleBinCullingCS_fail_unroll.hlsl

```diff
@@ -0,0 +1,229 @@
+// RUN: %dxc -E main -T cs_6_0 %s | FileCheck %s
+
+// CHECK: Could not unroll loop.
```

Comment:
Could you document why this file is expected to have a loop that fails unrolling? There's so much code in here that it doesn't immediately stand out.